gamemodes/terrortown/gamemode/client/cl_main.lua

```diff
@@ -195,7 +196,9 @@ function GM:InitPostEntity()
 		draw.CacheAvatar(plys[i]:SteamID64(), "medium") -- caching
 	end
 
-	timer.Create("cache_ents", 1, 0, GAMEMODE.DoCacheEnts)
+	timer.Create("cache_ents", 1, 0, function()
+		self:DoCacheEnts()
+	end)
```

Comment:
Is there a reason for defining this as an anonymous function?

Reply:
Because previously, it just called as `GM.DoCacheEnts`. This is a simple function reference WITHOUT referencing the GM / GAMEMODE. The problem is that the function in TTT requires GAMEMODE (it‘s written with `:`, not as a simple table function. Otherwise, self in this function wouldn‘t be GAMEMODE).